test/cli/package-test-simple/main_lib/__package.rb

```diff
@@ -5,4 +5,5 @@
 class Project::MainLib < PackageSpec
   import Project::Util
   test_import Project::TestOnly
+  export_for_test Project::MainLib::Lib
```

Comment:
If we're still open to bikeshedding the name, what about `test_export` for symmetry with `test_import`?

Reply:
The problem is that `test_export` isn't technically visible to tests in other packages. So it's not the opposite of `test_import`.